README.md

```diff
@@ -33,12 +33,17 @@ Prior running:
 ## Test the URL-Shortener Service
 
 * Run `curl -L -X POST 'http://localhost:8082/' -H 'Authorization: Bearer <JWT TOKEN HERE>' -H 'Content-Type: application/json' --data-raw '{"url":"http://www.nu.nl"}'` to shorten the URL and receive an ID.
-* Run `curl -L -X GET 'http://localhost:8082/<ID HERE>' -H 'Content-Type: application/json'` to get the shortened URL
```

Comment:
Maybe add another MD file for Demo NGINX reverse proxy. Now that the ports are omitted, it is assumed the NGINX server is running. The NGINX reverse proxy is a bonus. So now if someone follows the README, it will not find the url shortener service, as NGINX server is not running (yet)

Reply:
I think this should not been have removed anyway. as the RP demo is actually totally separated from it.

Reply:
Restored ports at curl commands in readme